src/review/routes/build/reducer.js

```diff
@@ -1,14 +1,19 @@
-// import actionTypes from 'redux/actionTypes';
+import actionTypes from 'redux/actionTypes';
 
-function reducer(state, action) {
+function buildReducer(state, action) {
   if (state === undefined) {
     state = {};
   }
 
   switch (action.type) {
+    case actionTypes.BUILD_FETCH_TASK:
```

Comment:
I'm planning adding a `watchTask` HoC, so having the progress state is going to help me out.